lua/harpoon/ui.lua

```diff
@@ -112,6 +112,13 @@ function M.toggle_quick_menu()
         "<Cmd>lua require('harpoon.ui').toggle_quick_menu()<CR>",
         { silent = true }
     )
+    vim.api.nvim_buf_set_keymap(
```

Comment:
Before I merge this, we should consider perhaps a better way here.

To me, updating the README to show custom menu closing may be better.
If we merge this, I may want to make it so that its behind a flag to turn on.  It seems a bit gotcha for me